Invalid escape sequence in top-level docstring

Hi,

the top-level docstring in bitstring v3.1.6 should be raw because it uses backslashes in its ASCII art diagram:
```
                      Bits (base class)
                     /    \
 + mutating methods /      \ + streaming methods
                   /        \
              BitArray   ConstBitStream
                   \        /
                    \      /
                     \    /
                    BitStream
```

Running pylint via pytest-pylint on code using bitstring (python 3.8 if that matters) results in a SyntaxError:
```
../../.envs/scientific/lib/python3.8/site-packages/_pytest/python.py:513: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
../../.envs/scientific/lib/python3.8/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../../.envs/scientific/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:152: in exec_module
    exec(co, module.__dict__)
test_codec.py:3: in <module>
    from bitstring import BitArray, CreationError
E     File "~/.envs/scientific/lib/python3.8/site-packages/bitstring.py", line 2
E       """
E       ^
E   SyntaxError: invalid escape sequence \
```
Making the docstring a raw one ("r" prefix) is fixing the issue.

By the way, nice job writing this library, it is my go to choice when dealing with serialization/deserialization protocols !

--- a/bitstring.py
+++ b/bitstring.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-"""
+r"""
 This package defines classes that simplify bit-wise creation, manipulation and
 interpretation of data.
 

